worker/provision_external_device.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "provision_external_device",
+    "description": "Submit device configuration settings",
```

Comment:
should we update the description after renaming and adding more tasks?

Reply:
Ooops, I missed that.